fizz_buzz_class.rb

```diff
@@ -0,0 +1,10 @@
+class FizzBuzzClass
```

Comment:
It would be better if this were a module instead of a class. Call the module `FizzbuzzHelper`

```ruby
module FizzBuzzHelper
  # Helper methods here!
end
```

Then in your main program, add this line
```ruby
include FizzBuzzHelper
```